lib/mpool/lib/mclass.h

```diff
@@ -109,23 +109,31 @@ mclass_upath(const struct media_class *mc);
  * @mc: mclass handle
  */
 int
-mclass_dirfd(struct media_class *mc);
+mclass_dirfd(const struct media_class *mc);
 
 /**
  * mclass_fset() - get mblock fileset handle
  *
  * @mc: mclass handle
  */
 struct mblock_fset *
-mclass_fset(struct media_class *mc);
+mclass_fset(const struct media_class *mc);
+
+/**
+ * mclass_ra_pages() - get read-ahead size in pages
```

Comment:
Don't repeat the function name in the comment. Will make a life that much easier whenever we decide to be consistent on comment style in the future.

Reply:
Just followed the pattern, will change.